dashboard/app/models/user.rb

```diff
@@ -958,6 +958,27 @@ def student_of?(teacher)
     teachers.include? teacher
   end
 
+  # @return {boolean} true if (1) Attended CSD TeacherCon '17 (2) are a CSD facilitator
+  def circuit_playground_pd_eligible?
+    csd_cohorts = %w(CSD-TeacherConPhiladelphia CSD-TeacherConPhoenix CSD-TeacherConHouston)
+
+    return true if cohorts.any? {|cohort| csd_cohorts.include?(cohort.name)}
```

Comment:
In the future, we think we're going to have a different table (supplied by @bencodeorg) that provides this data, but this should be sufficient until that table is ready.